test/e2e/cluster_config.sh

```diff
@@ -29,6 +29,7 @@ clusters:
     enable_rbac: "true"
     dynamodb_service_link_enabled: "false"
     skipper_ingress_cpu: 100m
+    skipper_default_filters: 'enableAccessLog(4,5) -> lifo(2000,20000,"3s") -> rfcHost()'
```

Comment:
This is already in config-defaults, is there a reason to duplicate it here?

Reply:
It's not needed if cluster/config-defaults.yaml are applied

Reply:
I had an impression that config defaults are not applied and therefore added `nlb_switch` [here](https://github.com/zalando-incubator/kubernetes-on-aws/pull/4651/files#diff-9a99d741a98cf62976440d87f64f503aab0266cd1e406365c9cd05c7ccdf012eR35)

Reply:
They're definitely applied, duplicating them would just cause them to go out of sync eventually, so I wouldn't do it without a reason.

Reply:
Removed. I guess there are quite some toggles to clean up then, like `kube_aws_ingress_controller_nlb_enabled`, `routegroups_validation`, `stackset_routegroup_support_enabled`, etc

Reply:
Regarding `nlb_switch`: it has `"false"` value in the else branch
https://github.com/zalando-incubator/kubernetes-on-aws/blob/9845fe22ca83bf164b8460dc5f8e85a184f111bb/cluster/config-defaults.yaml#L44-L52

I guess this could be solved differently using `.Cluster.Environment` like
https://github.com/zalando-incubator/kubernetes-on-aws/blob/9845fe22ca83bf164b8460dc5f8e85a184f111bb/cluster/config-defaults.yaml#L63-L67